build.default.props

```diff
@@ -104,16 +121,16 @@ behat.yml.template = ${behat.dir}/behat.yml.dist
 behat.yml.path = ${behat.dir}/behat.yml
 
 # The base URL to use in Behat tests.
-behat.base_url = http://web:8080
+behat.base_url = ${project.url.base}
```

Comment:
Duplicated bellow

Reply:
Back to localhost like it was in ssk: https://github.com/ec-europa/subsite-starterkit/blob/master/build.properties.dist#L228